src/pages/Route/components/CreateStep4/CreateStep4.tsx

```diff
@@ -44,7 +46,12 @@ const CreateStep4: React.FC<Props> = ({ form1, form2, redirect, ...rest }) => {
           <h2 style={style}>{formatMessage({ id: 'route.create.define.api.backend.server' })}</h2>
           <Step2 {...rest} form={form2} disabled />
           <h2 style={style}>{formatMessage({ id: 'route.create.plugin.configuration' })}</h2>
-          <PluginPage data={rest.data.step3Data.plugins} disabled />
+          {Boolean(Object.keys(plugins).length !== 0) && (
+            <PluginPage data={rest.data.step3Data.plugins} disabled />
+          )}
+          {Boolean(Object.keys(script).length !== 0) && (
+            <PluginChart data={rest.data.step3Data.script.chart} readonly onChange={() => {}} />
```

Comment:
what is `PluginChart`? do you mean `plugin orchestration`?

Reply:
PluginChart is a React component from `api7-dashboard/pluginchart` module, which is used to render chart according to the schema.

Reply:
it seems PluginOrchestration is a better choice, I will change`PluginChart` to `PluginOrchestration` later.